lib/logging-symbols.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = {
+  chef: '\uD83D\uDC69\u200D\uD83C\uDF73',
+  ticket: '\uD83C\uDF9F ',
+  crayon: '\uD83D\uDD8D ',
+  stop: '\uD83D\uDED1',
+  bomb: '\uD83D\uDCA3',
+  shield: '\uD83D\uDEE1 ',
+  shrug: '\uD83E\uDD37',
+  bathtub: '\uD83D\uDEC0',
+  bowAndArrow: '\uD83C\uDFF9',
+  bullseye: '\uD83C\uDFAF',
+}
```

Comment:
Any reason for defining this by hand rather than using a package?
Seems like we could pull in a package like [emojic](https://www.npmjs.com/package/emojic) (others are available, feel free to substitute your favourite) here and not have to 'maintain' this if we want to add more icons later.